docs/toggles.rst

```diff
@@ -1,47 +1,35 @@
-Toggles
+Plugins
 =======
 
-Toggles, also known as feature flags, allow limiting access to a set of functionality.
+There are a number of open source plugins which sit on top of the core CommCare functionality that enable a specific set of functionality.  These plugins are built and maintained by the CommCare open source community, thus are not included in any existing CommCare subscriptions on the Dimagi’s SaaS offering of CommCare.  Dimagi does provide explicit support for plugins for some customers under specific agreements.
 
-They are implemented as a couchdb-backed django app, designed to be *simple* and *fast* (automatically cached).
-
-Most toggles are configured by manually adding individual users or domains in the Feature Flags
-
-admin UI. These are defined by adding a new ``StaticToggle`` in this file. See ``PredictablyRandomToggle``
-
-and ``DynamicallyPredictablyRandomToggle`` if you need a toggle to be defined for a random subset
-
-of users.
+For self-hosted environments, plugins typically have a higher support burden and require directly reading the code to provide support. A smaller percentage of CommCare’s open source developer community typically has knowledge on any given plugin. If you are enabling plugins in your local environment, please make sure you have sufficient engineering expertise to be able to read direct code-level documentation. Plugins can be managed through the this `admin <UI https://www.commcarehq.org/hq/flags/?__hstc=240960668.9cbe5668b14d773d52583aa6ddd44168.1491228590522.1491493477971.1491498579156.19&__hssc=240960668.47.1491498579156&__hsfp=2069207928>`_.
```

Comment:
We should avoid ever linking to our production version of the website. 

Also - We should avoid saying "self-hosted environments," and simply say "When hosting the website, be aware that plugins aren't fully supported by the core committers and generally have a higher..."